### ISSUE TITLE
Neo4j browser 4.2.3 sucks

### ISSUE THREAD

**Issue by Mintactus** [neo4j/neo4j-browser#1311]
Sorry to be rude but

Everything about the recently new updated version of the Neo4j browser is wrong

- Deleting a query is longer
- Show and edit a favourite is longer
-  The query font text sucks for readability ( Most important point )

It's not great at all, it might be a matter of opinion, but I don't think so, something is really wrong with this update.

**Comment by HerrEmil** [comment]
@Mintactus The query font should still be Fira Code in the newer versions of Neo4j Browser. Could you inspect the query editor with devtools and check which font you are seeing now to help us find the problem?
I can see that the font weight was changed from 400 to 500 and this is visible in a side-by-side comparison with older versions. I have created a pull request #1314 with adjusted font weight and added fallback fonts in case Fira Code does not load. Is the issue you're seeing still visible after those changed?

**Comment by Mintactus** [comment]
Thank you

I tried to use it in a web browser with my chromebook but the http is
really heavy and slow now.
The elapsed time doesn't change anymore on the same frame when you run the
query again.

Thanks for your consideration but I just feel that this new release got out
too soon.

**Comment by OskarDamkjaer** [comment]
In the last update (4.2.5) we addressed the issues you've mentioned. We've updated the font to be more readable and made showing favorites just as quick like before while keeping our new features. As for the deleting part, I'm not 100% sure what you mean, but both deleting a query and a favorite is quick operations.

Deleting one or more favorites:
![Uploading CleanShot 2021-03-12 at 09.16.24.gif…]()

Deleting a query:
![CleanShot 2021-03-12 at 09 15 33](https://user-images.githubusercontent.com/10564538/110911873-8857ba00-8313-11eb-97ac-d67dc140ebbc.gif)

I've not been able to reproduce the "elapsed time" issue you've mentioned. I'll go ahead and close this issue. If you find any new issues or have any more concerns, drop a comment here and I'll re-open if needed :)
